src/order/boolean_algebra.lean

```diff
@@ -577,11 +577,13 @@ is_compl_bot_top.compl_eq
 @[simp] theorem compl_compl (x : α) : xᶜᶜ = x :=
 is_compl_compl.symm.compl_eq
 
-theorem compl_injective : function.injective (compl : α → α) :=
```

Comment:
It looks like it might be keeping this lemma around for readability.

Reply:
idk, do we often keep `_injective` and `_surjective` lemmas when `bijective` is available? I feel like it's just a waste of space, but don't have strong enough feelings about this to overrule you. do we make a `_surjective` lemma too, then?

Reply:
I find myself looking for `foo_injective` lemmas a lot in proofs, which means my search strategy doesn't work if we only have the stronger result `foo_bijective`. I rarely need the surjective version.

Another argument would be that `compl_injective` is apparently the most useful version to use in proofs from the rest of this patch, and removing it makes the diff bigger.

Neither is a particularly strong argument